CONTRIBUTING.md

```diff
@@ -70,7 +74,69 @@ will be posted in the issue's thread.
 * Bulleted points are fine.
 * Typically a hyphen or asterisk is used for the bullet, followed by a single space.
 
-### Merging Commits
+## Pull Request Process
+
+### Creating a Pull Request
+
+Use the pull request template to provide a complete description of the change.
+The template aims to capture important information to streamline the review
+process, ensure your changes are captured in release notes, and update related
+issues. Your pull request description and any discussion that follows is a
+contribution itself that will help the community and future contributors
+understand the project better.
+
+* Before submitting a pull request, please make sure you verify the changes
+  locally. The `Makefile` in this repository provides useful targets such as
+  `lint` and `test` to make verification easier.
+* Prefer small commits and small pull requests over large changes.
+  Small changes are easier to digest and get reviewed faster. If you find
+  that your change is getting large, break up your PR into small, logical
+  commits. Consider breaking up large PRs into smaller PRs, if each of them
+  can be useful on its own.
+* Have good commit messages. Please see [Commit Messages](#commit-messages)
+  section for guidelines.
+* Pull requests *should* reference an existing issue and include a `Fixes #NNNN`
+  or `Updates #NNNN` comment. Remember that `Fixes` will close the associated
+  issue, and `Updates` will link the PR to it.
+
+### Getting your Pull Request Reviewed, Approved, and Merged
+
+Once a pull request has been opened, the following must take place before it is merged:
+
+* It needs the `ok-to-merge` label
+* It needs to pass all the checks.
+* It needs to be approved by a [CODEOWNER](https://github.com/vmware-tanzu/tanzu-framework/blob/main/CODEOWNERS) for all files changed
+
+While these steps will not always take place in the same order, the following describes the process for a typical pull request once it is opened:
+
+1. Review is automatically requested from CODEOWNERS and `needs-triage` label is automatically added.
```

Comment:
Today only issues gets assigned "needs-triage" on creation, not PRs.
If the proposal is to now do the same for PRs, I would suggest that we don't.
Today the bit of PR triage we need to do is determine ok-to-merge or do-not-merge/x
I think documenting that's what triage means, and maybe explicitly mentioning that 
the lack of ok-to-merge/do-not-merge/x implies that PR has yet to be triaged might be sufficient.

Reply:
Oh, good point. I'll change this per your suggestion.

Reply:
pushed a new commit that removes reference to `needs-triage`. Left it separate so you can just check that diff, but can be squashed during merge, or I can squash and push again if preferred.